src/organisation/services/organisation.service.ts

```diff
@@ -1,27 +1,46 @@
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
 
-import { Observable, of } from 'rxjs';
+import { Observable, of, throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
 
+export const ENVIRONMENT = {
 
-const dummy = {
-  name: 'xyz solicitors Ltd',
-  address1: '10 Oxford Street',
-  town: 'London',
-  postcode: 'W1 ABC',
-
+  orgUri: '/api/organisation'
```

Comment:
should this not all be part of the config file ?

Reply:
yes will make this change :)